src/os/ui/ogc/ogclist.js

```diff
@@ -0,0 +1,256 @@
+goog.module('os.ui.ogc.List');
```

Comment:
The convention decided on in the ES6 docs is for these files/modules to be named things along the lines of `ListUI`. It is also preferred that the path and filenames match these module names as well as they can. If the file is titled ogclistui.js, the module should be `OGCListUI`

Reply:
Updated filenames and namespace names.